core/src/main/java/com/amplifyframework/predictions/models/FaceLivenessSession.kt

```diff
@@ -19,6 +19,8 @@ import com.amplifyframework.annotations.InternalAmplifyApi
 
 @InternalAmplifyApi
 class FaceLivenessSession(
+    val challengeId: String,
+    val challengeType: FaceLivenessChallengeType,
     val challenges: List<FaceLivenessSessionChallenge>,
```

Comment:
Not sure there is much we can do here without breaking API to liveness lib, but its a bit confusing that a "challenge" has "challenges" itself. It requires some context thought to the overall outer challenge and the inner challenges that make up the outer one.

Reply:
For sure, I had a bit of trouble ramping up to the code trying to figure out the difference. When we're planning to make breaking changes, I'd want to rename the inner "challenges" to like "challengeSteps" or something that doesn't include the word "challenge" to explicitly differentiate them.